### DISCUSSION TITLE
Any chance for support for later Anki builds?

### DISCUSSION THREAD

**Discussion by MetalFir** [AlexRiina/anki_cousins#11] [Ideas]
This is a really good add-on that's will give people alot of efficiency. But unfortunately may people have other addons that have not been updated therefore can not update their Anki to the latest version. I think the most common for compatibility is Anki 2.1.22.

**Comment by AlexRiina** [comment]
Hey @MetalFir, thanks for bringing that to my attention. I was probably a bit hasty in cutting official support for anything less than 2.1.35 in the latest release. I think some of the config access code changed around 2.1.24 but it shouldn't be hard to make the plugin consistent with 2.1.24+ and 2.1.22 config access methods. I'll try to install and test against 2.1.22 soon and then adjust the restriction on the plugin.

What's the addon that's keeping you at 2.1.22? I wasn't able to track down any usage data except the supported versions on the plugin list page and it seems like most plugins go up to 2.1.24 but may support higher versions (this plugin is listed as supporting up to 2.1.40 with the hope that the Anki internals don't change too much between 2.1.38 and then).

**Reply by MetalFir** [reply]
> Hey @MetalFir, thanks for bringing that to my attention. I was probably a bit hasty in cutting official support for anything less than 2.1.35 in the latest release. I think some of the config access code changed around 2.1.24 but it shouldn't be hard to make the plugin consistent with 2.1.24+ and 2.1.22 config access methods. I'll try to install and test against 2.1.22 soon and then adjust the restriction on the plugin.
> 
> What's the addon that's keeping you at 2.1.22? I wasn't able to track down any usage data except the supported versions on the plugin list page and it seems like most plugins go up to 2.1.24 but may support higher versions (this plugin is listed as supporting up to 2.1.40 with the hope that the Anki internals don't change too much between 2.1.38 and then).


The major addons that were keeping me from upgrading using was [AvgEase](https://ankiweb.net/shared/info/633293039) and [AutoLearningSteps](https://gist.github.com/eshapard/960f2508c2096ff663af3b6ebc925f16) but having a look around it seems like they've been updated by 3rd parties since I last checked so that's not an issue now. I'm going to see how far I can go in terms of Anki builds before they break.

And in terms of supported addons I think I got the info from [this reddit post](https://www.reddit.com/r/Anki/comments/jyv7eu/supported_addons_by_anki_version/) 

Edit: 2.1.26 is now when everything breaks for me

**Reply by AlexRiina** [reply]
Thanks, those look like really interesting plugins. I did a little audit of my own cards based on the AvgEase plugin and I'm probably wasting some time over-reviewing on some of my decks.

I've just released a small update that hopefully restores compatibility with 2.1.22. LMK if you run into any issues with the new version (based on https://github.com/AlexRiina/anki_cousins/pull/12)
